src/js/extend.js

```diff
@@ -3,38 +3,7 @@
  * @module extend
  */
 
-/**
- * A combination of node inherits and babel's inherits (after transpile).
- * Both work the same but node adds `super_` to the subClass
- * and Bable adds the superClass as __proto__. Both seem useful.
- *
- * @param {Object} subClass
- *        The class to inherit to
- *
- * @param {Object} superClass
- *        The class to inherit from
- *
- * @private
- */
-const _inherits = function(subClass, superClass) {
-  if (typeof superClass !== 'function' && superClass !== null) {
-    throw new TypeError('Super expression must either be null or a function, not ' + typeof superClass);
-  }
-
-  subClass.prototype = Object.create(superClass && superClass.prototype, {
-    constructor: {
-      value: subClass,
-      enumerable: false,
-      writable: true,
-      configurable: true
-    }
-  });
-
-  if (superClass) {
-    // node
-    subClass.super_ = superClass;
-  }
-};
+import _inherits from '@babel/runtime/helpers/inherits';
```

Comment:
_inherits from babel is exactly the same as our code.

Reply:
Since babel includes the  `_inherits` from `@babel/runtime/helpers/inherits` in our code regardless, we may as well not include it twice

Reply:
This change broke a plugin https://github.com/videojs/video.js/issues/6328.